src/plugins/imageZoom/styles.css

```
@@ -29,3 +29,10 @@
     top: 50%;
     transform: translateY(-50%);
 }
+
+[class^="focusLock"] > [class^="carouselModal"] > [class*="modalCarouselWrapper"] > [class^="wrapper"] {
```

this seems to work too?
```suggestion
#magnify-modal {
```